uw-frame-components/css/buckyless/general.less

```diff
@@ -5,7 +5,7 @@ body {
 }
 a {
   transition: @link-hover-transition;
-  &:not(.md-button), &:not(.btn) {
```

Comment:
Yeah, it was weird. The way it breaks down is like its two different selectors.  Like 
```
a:not(.md-button){
 //some style
}

a:not(.btn) {
  //some style
}
```